index.html

```diff
@@ -1,14 +1,164 @@
 <!DOCTYPE HTML>
 <html>
   <head>
-    <title>Mockup Home</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <title>Remock Project</title>
+    <style>
+      @import "index.css";
+    </style>
   </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+
+  <body class="body1">
+
+    <section id="header">
+        <h1>HOME PAGE</h1> 
+        <H6>THE PAGE FOR THE HOME</H6>
+    </section>
+
+      <div class="content">
+        <h2> TAKE A LOOK AT THIS AMAZING <strong>HOME PAGE!</strong></h2>
+      </div>
+      <div id="homeimgs">
+        <div class="img1wrapper">
+          <img class="img1" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/Tree_of_life.jpg"/>
+          <div>IMAGE ONE</div>
+        </div>
+        <div class="img2wrapper">
+          <img class="img2" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/grass.jpg"/>
+          <div>IMAGE TWO</div>
+        </div>
+        <div class="img3wrapper">
+          <img class="img3" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/glacier_water.jpg"/>
+          <div>IMAGE THREE
+          </div>
+        </div>
+      </div>

```

Comment:
All of these images are located on your computer. Users will not have access to them unless they are being hosted online somewhere and linked to their web locations.

Reply:
I have switched the images to all online sourced images (picsum).